packages/components/src/public/QueryModel/CustomizeGridViewModal.spec.tsx

```diff
@@ -573,3 +574,38 @@ describe('ColumnChoiceGroup', () => {
         wrapper.unmount();
     });
 });
+
+describe('includedColumnsForCustomizationFilter', () => {
+    test('hidden', () => {
+        let col = QueryColumn.create({ name: 'testColumn', hidden: false });
+        expect(includedColumnsForCustomizationFilter(col, false)).toBeTruthy;
```

Comment:
If you reordered these you would only need to reset `col` once.

Reply:
done